python/ray/autoscaler/autoscaler.py

```diff
@@ -130,32 +141,27 @@
     # Map of remote paths to local paths, e.g. {"/tmp/data": "/my/local/data"}
     "file_mounts": (dict, OPTIONAL),
 
-    # List of commands that will be run before `setup_commands`. If docker is
-    # enabled, these commands will run outside the container and before docker
-    # is setup.
-    "initialization_commands": (list, OPTIONAL),
-
-    # List of common shell commands to run to setup nodes.
-    "setup_commands": (list, OPTIONAL),
-
-    # Commands that will be run on the head node after common setup.
-    "head_setup_commands": (list, OPTIONAL),
+    # Commands that will be run when the cluster is first created.
+    "setup_commands": COMMANDS_SCHEMA,
```

Comment:
Should we call these:
- `node_setup_commands`
- `node_restart_commands`
- `ray_restart_commands`
?

Reply:
Good suggestion, changed.